server/stycoon/command_move.go

```diff
@@ -0,0 +1,53 @@
+package stycoon
+
+import (
+	"fmt"
+
+	"github.com/gdg-garage/space-tycoon/server/database"
+)
+
+func NewMoveCommand(command Command) *MoveCommand {
+	return &MoveCommand{
+		Type:        command.Type,
+		Destination: command.Destination,
+	}
+}
+
+func (game *Game) CreateWaypointFromCoordinates(c []int64) (int64, error) {
+	if len(c) != 2 {
+		return 0, fmt.Errorf("move command: coordinates must have exactly two elements")
+	}
+	// TODO search for existing waypoints first
+	waypointId, err := database.InsertObject(game.db, c[0], c[1])
+	if err != nil {
+		return 0, err
+	}
+	database.InsertWaypoint(game.db, waypointId)
```

Comment:
remove

Reply:
can u explain more?

Reply:
u mean, insert object does the job already?

Reply:
the table has been removed

Reply:
ok. good to know